trust-sdk/trust-sdk-specs.md

```diff
@@ -53,48 +78,62 @@ lrzEHpAAIAmoB0Yegc2ZqxtkSkXlYo_TEg4eBrjopGUj9ySxJh6JlfToGqR7yNKzV8cD_yN_jVR5YrVa
 7Tlfb_0eV47yG1Co&id=4
 ```
 
-### Sign Message
+### Sign simple transaction
+
+This command is the simplified version of `sdk_sign`, it accepts a simple `Transaction` object instead of heavier protobuf model
 
 #### Command
-* Command: `sdk_sign_message`
+
+* Command: `sdk_transaction`
 * Parameters:
   * `coin`: BIP44 coin code
-  * `data`: message to sign, caller must encode or (hash it) as hex string
-  * `meta`: dapp metadata. This attribute is optional and encoded as a dictionary. Check how it's encoded in [Dictionary Encoding](#dictionary-encoding).
+  * `to`: recipient address
+  * `amount`: amount in human-readable (unit) format
+  * `action`: flag that indicates if the wallet should broadcast the transaction. Valid values: `send`, `sign`.
+  * `token_id`: (Optional) token id, follows standard of unique identifier on the blockhain as smart contract address or asset ID
```

Comment:
in deeplink we construct from coin and token_id
* `asset`: Universal asset identifier